assets/src/styles/blocks/PageHeader.scss

```diff
@@ -0,0 +1,107 @@
+.is-pattern-p4_page-header {
+  margin-inline-start: calc(((100vw - 100%) / 2) * -1);
+  padding: 0;
+  width: 100vw;
+
+  img.wp-block-cover__image-background {
+    height: inherit;
+    min-width: 360px;
+    max-height: 270px;
+  }
+
+  .wp-block-group {
+    padding-left: 24px;
+    padding-right: 24px;
+  }
+
+  .wp-block-group:first-child {
+    display: block;
+    margin-top: 233px;
+    max-width: 520px;
+  }
+
+  h1.has-background,
+  h2.has-background {
+    display: inline;
+    font-size: 2.5rem;
+    font-weight: 700;
+    letter-spacing: -1px;
+    line-height: initial;
+    padding: 0;
+    white-space: pre-wrap;
+  }
+
+  .wp-block-group:nth-child(2) {
+    display: block;
+  }
+
+  .wp-block-group:last-child {
+    margin-bottom: 51px;
+  }
+
+  p {
+    font-size: 1.25rem;
```

Comment:
It seems you're not changing this font size across screen sizes, so you could use block settings for it